en/editors_and_tools/creator_to_cocos2dx.md

```diff
@@ -21,27 +22,22 @@ Adding C++ and Lua language support to __Cocos Creator__ is easy:
 
     ![](creator_to_cocos2dx-img/folder_structure.png "directory structure")
 
-  In the __Project__ menu inside __Creator__ a new menu option will appear
-  __LuaCPP Support__.
+    In the __Project__ menu inside __Creator__ a new menu option will appear __LuaCPP Support__.
 
     ![](creator_to_cocos2dx-img/project_menu.png "project menu")
 
 ### Plugin Setup
 To run the plugin:
 
-* select __Project__ -> __LuaCPP Support__ -> __Setup Target Project__.
-
-  ![](creator_to_cocos2dx-img/dialog_options.png "dialog options")
+* select __Project__ -> __LuaCPP Support__ -> __Setup Target Project__. It is required to tell __Cocos Creator__ where to build all the necessary files.
 
-  It is required to tell __Cocos Creator__ where to build all the necessary files.
+    ![](creator_to_cocos2dx-img/dialog_options.png "dialog options")
 
 * select __Build__.
 
-  ![](creator_to_cocos2dx-img/dialog_options.png "dialog options")
-
 * use the resulting dialog box to set the build options that you need.
```

Comment:
when I try to use this plugin with cocos2d-x 3.16 and Creator 1.6.2, this step isn't here no longer. @minggo said this have been removed in new vision, should we remove this step in English docs?

Reply:
What step? Just the picture? __Build__ is still required.

Reply:
`* use the resulting dialog box to set the build options that you need.`
 this step `build options`

Reply:
__build__ is required, __build options__ not

Reply:
Done.